language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java

```diff
@@ -136,7 +138,7 @@ public static void printSyntax(PrintStream out, List<Token> tokens) {
   /**
    * Connects to the Natural Language API using Application Default Credentials.
    */
-  public static CloudNaturalLanguageAPI getLanguageService() 
+  public static CloudNaturalLanguageAPI getLanguageService()
     throws IOException, GeneralSecurityException {
     GoogleCredential credential =
         GoogleCredential.getApplicationDefault().createScoped(CloudNaturalLanguageAPIScopes.all());
```

Comment:
@gguuss You need to make changes to printSyntax() to add more tokens returned in v1.